src/components/Details.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure code quality.

Reply:
Hello @rloterh, thanks for the suggestion! I would appreciate it very much if you helped me solve this one, how would you suggest that I fix my code so that it complies with this linter without reducing functionality? Thank you very much in advance.

Reply:
Never mind, got it. Thanks for pointing it out.